debian/control

```diff
@@ -50,7 +49,7 @@ Build-Depends: automake,
                libldap2-dev,
                libleveldb-dev,
                liblttng-ust-dev,
-               liblua5.3-dev,
+               liblua5.3-dev | liblua5.4-dev,
```

Comment:
why is this needed, both ubuntu focal and bionic use liblua5.3 ?

Reply:
yeah, i am scratching my own itch. as on debian sid, quite a few packages are already compiled against liblua5.4. for instance, see https://packages.debian.org/sid/neomutt . so i didn't want to install liblua5.3-dev for compiling ceph since i've installed liblua5.4-dev. if you are still concerned about this change, i will be happy to drop it.

Reply:
from thew lua RGW binding side of things i don't think it makes a difference - lua5.4 is backward compatible, and [the changes](https://www.lua.org/manual/5.4/readme.html#changes) should not impact it anyway.
but I'm not 100% regarding luarocks (which is also our dependency) which has a lua5.3 dependency even in its [latest version](https://packages.debian.org/sid/luarocks).

this could be easily tested, even without the RGW - just make sure that luarocks work on a debian sid machine with lua5.4 installed on.

Reply:
> this could be easily tested, even without the RGW - just make sure that luarocks work on a debian sid machine with lua5.4 installed on.

```
pi   --\ luarocks                                                                                                                                                                                                                        +705 kB   <none>         2.4.2+dfsg-1.1
  --\ Depends (6)
    --- liblua5.1-dev | liblua5.2-dev | liblua5.3-dev
    --- lua-any
    --\ lua5.1 | lua5.2 | lua5.3
p     lua5.1 5.1.5-8.1+b3
i     lua5.2 5.2.4-1.1+b3
p     lua5.3 5.3.3-1.1+b1
```

ahh, then i have to drop this change.